controls/stig_rhel9.yml

```diff
@@ -4124,22 +4124,6 @@ controls:
             - configure_crypto_policy
         status: automated
 
-    -   id: RHEL-09-672035
-        levels:
-            - medium
-        title: RHEL 9 must implement DOD-approved encryption in the OpenSSL package.
-        rules:
-            - configure_openssl_crypto_policy
-        status: automated
-
-    -   id: RHEL-09-672040
-        levels:
-            - medium
-        title: RHEL 9 must implement DOD-approved TLS encryption in the OpenSSL package.
-        rules:
-            - configure_openssl_tls_crypto_policy
```

Comment:
You have removed this rule from the RHEL 9 data stream. You need to add this rule to the RHEL 9 default profile.